Band.h

```diff
@@ -0,0 +1,18 @@
+struct Band
+{
+    Wurlitzer keys;
+    ElectricBass bass;
```

Comment:
where is the `#pragma once`?
Where are the `#include` statements for these 2 types?
- [ ] revise